src/reducers/map.js

```diff
@@ -0,0 +1,15 @@
+import { MAP_UPDATE_PAN } from 'constants/map';
+
+// TODO: complete map initial state
+const initialState = {
+  zoom: 1
+};
+
+export default function (state = initialState, action) {
+  switch (action.type) {
+    case MAP_UPDATE_PAN:
+      return Object.assign({}, state, { zoom: action.payload.zoom });
```

Comment:
As long as we are going to pass the lat, lng and zoom you should change the payload